src/test/java/unstable/RepositorySelectorTest.java

```diff
@@ -100,16 +100,39 @@ public void repositorySelectorTest() {
         assertEquals(4, comboBox.getItems().size());
         assertEquals("dummy4/dummy4", primaryRepo);
 
+        // we check if deleting used repo does not remove it
+        click("Repos");
+        push(KeyCode.DOWN);
+        push(KeyCode.RIGHT);
+        push(KeyCode.DOWN);
+        push(KeyCode.DOWN);
+        push(KeyCode.DOWN); // first used repo
+        push(KeyCode.ENTER);
+        PlatformEx.waitOnFxThread();
+        assertEquals(4, comboBox.getItems().size());
+
         // we check if delete repo works
+        click("Boards"); // trigger clicking "Repos"
```

Comment:
This and below is to trigger the next simulations. Without this, some of them are just skipped. It went undetected since it passes the `assert`. Seems that the framework has a problem with using the same node again (this is also evident in `KeyboardShortcutTests`).